geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/util/RasterUtils.scala

```diff
@@ -200,7 +212,6 @@ object RasterUtils {
 
   def bufferCrop(src: BufferedImage, ulx: Int, uly: Int, w: Int, h: Int): BufferedImage = {
     val result = src.getSubimage(ulx, uly, w, h)
-    src.flush()
```

Comment:
I read something and it led me to believe they may not be helping / may be bad to do. I want to look into this a bit more to figure out the wise and good thing.